### PR TITLE
[RAPPS] Improve Turkish (tr-TR) translation

### PR DESCRIPTION
**Pull request by scropool**
I changed english lines to turkish and used correct translations because im turkish

### CODE REVIEW

**Comment by scropool** [comment]
Cc @reactos/lang-turkish

**Comment by binarymaster** [comment]
Cc @reactos/lang-turkish

File: base/applications/rapps/lang/tr-TR.rc [hunk 1]
```diff
@@ -273,5 +274,5 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_INSTGEN_CONFIRMUNINST "Are you sure you want to uninstall %s?"
+    IDS_INSTGEN_CONFIRMUNINST "Uygulamayı kaldırmak istediğinize emin misiniz: \"%s\"?"
```
Comment (binarymaster) [inline base/applications/rapps/lang/tr-TR.rc:277]:
```suggestion
    IDS_INSTGEN_CONFIRMUNINST "Uygulamayı kaldırmak istediğinize emin misiniz: ""%s""?"
```
Double quotes in RC files are escaped like that ^

**Comment by binarymaster** [comment]
Also cc @eersoy93 @Zaryob for review.

File: base/applications/rapps/lang/tr-TR.rc [hunk 2]
```diff
@@ -17,17 +18,17 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "&Çıkış", ID_EXIT
     END
-    POPUP "Y&azılım"
+    POPUP "&Yazılım"
```
Comment (HBelusca) [inline base/applications/rapps/lang/tr-TR.rc:21]: You've now introduced an accelerator collision with the next menu, `"&Yardım`, see below.

File: base/applications/rapps/lang/tr-TR.rc [hunk 3]
```diff
@@ -17,17 +18,17 @@ BEGIN
...
     BEGIN
         MENUITEM "&Kur\tCtrl+Enter", ID_INSTALL
-        MENUITEM "K&aldır\tCtrl+Del", ID_UNINSTALL
+        MENUITEM "&Kaldır\tCtrl+Del", ID_UNINSTALL
```
Comment (HBelusca) [inline base/applications/rapps/lang/tr-TR.rc:24]: There's now an accelerator collision with the previous menu item `&Kur`

File: base/applications/rapps/lang/tr-TR.rc [hunk 4]
```diff
@@ -41,7 +42,7 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM "&Bağlantıyı Tarayıcıda Aç", ID_OPEN_LINK
-        MENUITEM "Bağlantıyı Panoya &Kopyala", ID_COPY_LINK
+        MENUITEM "&Bağlantıyı Panoya Kopyala", ID_COPY_LINK
```
Comment (HBelusca) [inline base/applications/rapps/lang/tr-TR.rc:45]: Same comment

File: base/applications/rapps/lang/tr-TR.rc [hunk 5]
```diff
@@ -50,14 +51,14 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM "&Kur\tCtrl+Enter", ID_INSTALL
-        MENUITEM "K&aldır\tCtrl+Del", ID_UNINSTALL
+        MENUITEM "&Kaldır\tCtrl+Del", ID_UNINSTALL
```
Comment (HBelusca) [inline base/applications/rapps/lang/tr-TR.rc:54]: Same comment

File: base/applications/rapps/lang/tr-TR.rc [hunk 6]
```diff
@@ -239,7 +240,7 @@ STRINGTABLE
 BEGIN
     IDS_LICENSE_OPENSOURCE "Açık Kaynak"
     IDS_LICENSE_FREEWARE "Ücretsiz"
-    IDS_LICENSE_TRIAL "Deneme/Tanıtım"
+    IDS_LICENSE_TRIAL "Deneme Sürümü"
```
Comment (HBelusca) [inline base/applications/rapps/lang/tr-TR.rc:243]: The corresponding English translation is `IDS_LICENSE_TRIAL "Trial/Demo"`
so, something similar translated in Türkish would be preferable.

File: base/applications/rapps/lang/tr-TR.rc [hunk 7]
```diff
@@ -69,7 +70,7 @@ BEGIN
     GROUPBOX "Genel", -1, 4, 2, 240, 61
     AUTOCHECKBOX "&Pencere Konumunu Kaydet", IDC_SAVE_WINDOW_POS, 15, 12, 219, 12
     AUTOCHECKBOX "&Başlangıçta Erişilebilir Programların Listesini Yenile", IDC_UPDATE_AVLIST, 15, 29, 222, 12
-    AUTOCHECKBOX "P&rogramların Kurulma ve Kaldırma Günlüğünü Sakla", IDC_LOG_ENABLED, 15, 46, 219, 12
+    AUTOCHECKBOX "&Programların Kurulma ve Kaldırma Günlüğünü Sakla", IDC_LOG_ENABLED, 15, 46, 219, 12
```
Comment (HBelusca) [inline base/applications/rapps/lang/tr-TR.rc:73]: Now there's an accelerator collision with the first checkbox "&Pencere Konumunu Kaydet"

**Review by HBelusca — CHANGES_REQUESTED** [review]

File: base/applications/rapps/lang/tr-TR.rc (hunk 6 above)
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc:243]: I wrote it this way to make it more meaningful in Turkish so there is no mistake on turkish

File: base/applications/rapps/lang/tr-TR.rc (hunk 2 above)
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc:21]: what does accelerator collison means ?

Comment (HBelusca) [reply base/applications/rapps/lang/tr-TR.rc:21]: The `&` thingie. See for example:
https://reactos.org/wiki/Translation_Introduction#Common_Characters
https://learn.microsoft.com/en-us/windows/win32/menurc/about-menus#menu-access-keys

Comment (SigmaTel71) [reply base/applications/rapps/lang/tr-TR.rc:21]: You have single key combination for multiple menu actions, in this case pressing Alt+Y will choose either `Yazılım` or `Yardım`, depending on what is earlier in the menu.

File: base/applications/rapps/lang/tr-TR.rc (hunk 4 above)
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc]: solved.

File: base/applications/rapps/lang/tr-TR.rc [hunk 8]
```diff
@@ -27,7 +28,7 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "&Yenile\tF5", ID_REFRESH
         MENUITEM SEPARATOR
-        MENUITEM "Uygulama Veri Ta&banını Yenile\tCtrl+F5", ID_RESETDB
+        MENUITEM "Uygulama Veri Tabanını Yenile\tCtrl+F5", ID_RESETDB
```
Comment (JoachimHenze) [inline base/applications/rapps/lang/tr-TR.rc:31]: No, don't strip that accelerator!

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

File: base/applications/rapps/lang/tr-TR.rc (hunk 8 above)
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc:31]: look at my new commits I fixed the accelerator stages

> No, don't strip that accelerator!

Comment (JoachimHenze) [reply base/applications/rapps/lang/tr-TR.rc:31]: No you still didn't fix that.

File: base/applications/rapps/lang/tr-TR.rc (hunk 6 above)
Comment (JoachimHenze) [reply base/applications/rapps/lang/tr-TR.rc:243]: Unresolved also. What Hermes complained about was the missing slash between the 2 words.

Comment (binarymaster) [reply base/applications/rapps/lang/tr-TR.rc:243]: [IDS_LICENSE_TRIAL](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=IDS_LICENSE_TRIAL) seems to be a bit controversial, see ru-RU and uk-UA there.

Comment (JoachimHenze) [reply base/applications/rapps/lang/tr-TR.rc:243]: Well, maybe those could be improved too, outside of this PR. You can judge that better as a native speaker @binarymaster .

Comment (JoachimHenze) [reply base/applications/rapps/lang/tr-TR.rc:243]: Okay, the google translator gave me more insight in scropools change:
image

He stripped the 2nd term. Is ok for me then.

Comment (binarymaster) [reply base/applications/rapps/lang/tr-TR.rc:243]: Ok, in this specific tr-TR case I would suggest either `Deneme Sürümü/Tanıtım` or `Deneme/Tanıtım Sürümü`.

File: base/applications/rapps/lang/tr-TR.rc (hunk 8 above)
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc]: oh sorry my bad i changed now and changed deneme/tanıtım sürümü line in latest commit

**Comment by binarymaster** [comment]
@vsaucing could you please also review this PR, since we have so few Turkish translators active.

**Review by vsaucing — APPROVED** [review]
okay this looks right

**Review by Zaryob — CHANGES_REQUESTED** [review]

File: base/applications/rapps/lang/tr-TR.rc [hunk 9]
```diff
@@ -205,13 +206,13 @@ BEGIN
     IDS_APP_REG_REMOVE "Kurulan programın girdiyi kayıt defterinden silmek istediğinizden emin misiniz?"
     IDS_INFORMATION "Bilgi"
     IDS_UNABLE_TO_DOWNLOAD "Paket indirilemiyor! Adres bulunamadı!"
-    IDS_UNABLE_TO_DOWNLOAD2 "Paket indirilemiyor! İnternet bağlantınızı denetleyiniz!"
```
[the remark this answers is not in the document]
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc:208]: to control is more meaningful in Turkish than to supervise which means something different

File: base/applications/rapps/lang/tr-TR.rc [hunk 10]
```diff
@@ -193,7 +194,7 @@ BEGIN
     IDS_SEARCH_TEXT "Ara"
     IDS_APPS_COUNT "Uygulama Sayısı: %d; Seçili: %d"
     IDS_WELCOME_TITLE "ReactOS Uygulama Yöneticisi'ne hoş geldiniz.\n\n"
-    IDS_WELCOME_TEXT "Solda bir kategori seçiniz, ardından kurmak ya da kaldırmak için bir uygulama seçiniz.\nReactOS'un İnternet sitesi: "
+    IDS_WELCOME_TEXT "Soldaki Menüden bir kategori seçiniz, ardından kurmak ya da kaldırmak için bir uygulama seçiniz.\nReactOS'un İnternet sitesi: "
```
[the remark this answers is not in the document]
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc]: sure

File: base/applications/rapps/lang/tr-TR.rc [hunk 11]
```diff
@@ -205,13 +206,13 @@ BEGIN
     IDS_APP_REG_REMOVE "Kurulan programın girdiyi kayıt defterinden silmek istediğinizden emin misiniz?"
     IDS_INFORMATION "Bilgi"
     IDS_UNABLE_TO_DOWNLOAD "Paket indirilemiyor! Adres bulunamadı!"
-    IDS_UNABLE_TO_DOWNLOAD2 "Paket indirilemiyor! İnternet bağlantınızı denetleyiniz!"
+    IDS_UNABLE_TO_DOWNLOAD2 "Paket indirilemiyor! İnternet bağlantınızı kontrol ediniz!"
     IDS_UNABLE_TO_REMOVE "Programın girdisi kayıt defterinden silinemiyor."
     IDS_UNABLE_TO_INSTALL "Yükleyici açılamıyor!"
     IDS_UNABLE_TO_QUERY_CERT "Sertifika bilgisi alınamıyor.\n\nYine de devam edilsin mi?"
     IDS_INTEG_CHECK_TITLE "Paket bütünlüğü doğrulanıyor…"
-    IDS_INTEG_CHECK_FAIL "Paket bütünlük denetimini geçmedi, bozulmuş veya indirme esnasında oynanmış olabilir. Yazılımı çalıştırmanız önerilmez."
```
[the remark this answers is not in the document]
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc:213]: sure i putted back

File: base/applications/rapps/lang/tr-TR.rc [hunk 12]
```diff
@@ -17,17 +18,17 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "&Çıkış", ID_EXIT
     END
-    POPUP "Y&azılım"
```
[the remark this answers is not in the document]
Comment (scropool) [reply base/applications/rapps/lang/tr-TR.rc]: > Not sure about this change. & is bot changes so much…

so what change can I make here?

**Comment by vsaucing** [comment]
Everything is right except line 276.

**Review by HBelusca — APPROVED** [review]

File: base/applications/rapps/lang/tr-TR.rc [hunk 13]
```diff
@@ -273,5 +274,5 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_INSTGEN_CONFIRMUNINST "Are you sure you want to uninstall %s?"
+    IDS_INSTGEN_CONFIRMUNINST "Uygulamayı kaldırmak istediğinize emin misiniz: ""%s""?"
```
Comment (HBelusca) [inline base/applications/rapps/lang/tr-TR.rc:277]: Should this like be changed or improved?

**Comment by binarymaster** [comment]
Why closed?

**Comment by vsaucing** [comment]
?

**Pull request closed without merging**